cmd/content-sources/main.go

```diff
@@ -45,6 +46,11 @@ func main() {
 	}
 	defer db.Close()
 
+	err = dao.SetupGormTable(db.DB)
+	if err != nil {
+		log.Fatal().Err(err).Msg("Failed to setup gorm table.")
+	}
+
```

Comment:
This is needed to use soft delete on a joined table i.e. "template_repository_configurations". Not sure if this is the best way to handle this, but it's something you just run once. I feel like it's easy to forget about this way.

Reply:
would it make sense to run it in db.Connect() ?

Reply:
It does make sense, but that causes a circular dependency error between db and models

Reply:
ah i see, this is tricky :/   

one thing that might simplify it is to have the method call be fatal, maybe rename it like SetupGormTableOrFail  to make it explicit